WooCommerce/Classes/POS/Controllers/PointOfSaleItemsController.swift

```diff
@@ -31,6 +33,16 @@ protocol PointOfSaleItemsControllerProtocol {
 
     @MainActor
     func loadItems(base: ItemListBaseItem) async {
+        setLoadingState(base: base)
+        await loadFirstPage(base: base)
+    }
+
+    @MainActor func refreshItems(base: ItemListBaseItem) async {
+        await loadFirstPage(base: base)
```

Comment:
super nit: just to be consistent

```suggestion
    @MainActor
    func refreshItems(base: ItemListBaseItem) async {
        await loadFirstPage(base: base)
```